scripts/zones/Xarcabard/npcs/qm2.lua

```diff
@@ -24,8 +25,8 @@ end;
 -----------------------------------
 
 function onTrigger(player,npc)
-    local BorealTiger = GetMobAction(17236204);
-    if ((OldSchoolG2 == false) or (BorealTiger == ACTION_NONE or BorealTiger == ACTION_SPAWN)) then
+    local borealState = GetMobAction(17236204);
+    if OldSchoolG2 == false or borealState == ACTION_DEATH or borealState == ACTION_FADE_OUT or borealState == ACTION_NONE then
```

Comment:
psst, we're almost all include the parentheses for readability ppl here. (ignoring what kj already pointed out)